k8s/aat/common/neuvector/storage-secret.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: bitnami.com/v1alpha1
 kind: SealedSecret
 metadata:
   creationTimestamp: null
-  name: storage-secret
+  name: nv-storage-secret
```

Comment:
Do you need to change the name of the file too? otherwise we'll duplicate config when rotating sealed secret keys